components/PriortyBillsCard/PriorityBillsCard.tsx

```diff
@@ -0,0 +1,59 @@
+import { Card } from "react-bootstrap"
+import Styles from "./PriorityBillsCard.module.css"
+
+type bill = {
+  id: string
+  billNumber: string
+  title: string
+  approval: string
+}
+
+export const PriorityBillsCard = (props: {
+  bills: bill[]
+  bill_id: string
```

Comment:
I would rename this selectedBillId, so it's clearer what this controls. Similarly, we should change onClick to `onBillSelected: (billId: string) => void`.